### PR TITLE
RDM-5566 - results ordering defaults for workbasket and search results.

### PR DESCRIPTION
**Pull request by smathangi**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDM-5566
https://tools.hmcts.net/jira/browse/RDM-3974

### Change description ###

default sort ordering for workbasket and search results.
https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1126761193&__ncforminfo=KaHlLtxesA35Kf0Gqp8uwsUpWdlMJ92fAjenZ7T77Hgjc06MpmVx5TYo7zABc6XRNF90fwDNPvLtZmH9oDRFc-FDRaFVd6FO

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: dependency-check-suppressions.xml [hunk 1]
```diff
@@ -210,5 +210,14 @@
         <cve>CVE-2015-5377</cve>
         <cve>CVE-2015-5531</cve>
         <cve>CVE-2019-7611</cve>
+        <cve>CVE-2019-7614</cve>
+    </suppress>
+    <suppress>
```
Comment (smathangi) [inline dependency-check-suppressions.xml]: failing on master as well.. so temporarily ignored. will revert this once it's fixed on master.

File: excel-importer/src/test/java/uk/gov/hmcts/ccd/definition/store/excel/parser/SearchInputLayoutParserTest.java [hunk 2]
```diff
@@ -61,4 +61,13 @@ public void shouldFailWhenNoDataInWorksheet() {
     public void shouldReturnNameWhenAsked() {
         assertThat(classUnderTest.getLayoutName(), is("Search Inputs"));
     }
+
+    @DisplayName("Should fail when populateSortOrder is invoked")
```
Comment (rafalkalita) [inline excel-importer/src/test/java/uk/gov/hmcts/ccd/definition/store/excel/parser/SearchInputLayoutParserTest.java]: Please add @Test and fix the test.

Comment (smathangi) [reply excel-importer/src/test/java/uk/gov/hmcts/ccd/definition/store/excel/parser/SearchInputLayoutParserTest.java]: Good spot. My bad 😄  fixed now.

File: excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/parser/GenericLayoutParser.java [hunk 3]
```diff
@@ -138,10 +151,76 @@ private UserRoleEntity getRoleEntity(GenericLayoutEntity layoutEntity, String sh
                 userRole, sheetName, layoutEntity.getCaseField().getReference())));
     }
 
+    private void validateSortOrders(List<DefinitionDataItem> layoutItems, CaseTypeEntity caseType) {
+        List<DefinitionDataItem> sortDataItems = layoutItems.stream()
+            .filter(ddi -> StringUtils.isNotEmpty(ddi.getString(RESULTS_ORDERING)))
+            .collect(Collectors.toList());
+        if (CollectionUtils.isNotEmpty(sortDataItems)) {
```
Comment (smathangi) [inline excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/parser/GenericLayoutParser.java]: worth to extract to a separate validator at this stage?

**Pull request merged**